gfx_scene.h

```diff
@@ -1057,7 +1113,7 @@ class GfxSceneInternal
                         if(std::get<2>(key) >= 0)
                         {
                             vertex.uv.x = obj_reader.GetAttrib().texcoords[2 * std::get<2>(key) + 0];
-                            vertex.uv.y = obj_reader.GetAttrib().texcoords[2 * std::get<2>(key) + 1];
+                            vertex.uv.y = 1.0f - obj_reader.GetAttrib().texcoords[2 * std::get<2>(key) + 1];
```

Comment:
flipped the uvs in .obj files to allow them to be used with the default image loading